shared/components/splash-example.jsx

```diff
@@ -59,7 +59,7 @@ export default React.createClass({
                             # Photos:
                         </span>
 
-                        <select
+                        <select className="num-photos-select"
```

Comment:
This was added only for testing purposes to have a hook. It was the most expedited way to get to the DOM node. Also, it adds symmetry with the locale select component.